packages/utils/src/route.js

```diff
@@ -173,6 +174,18 @@ export const createRoutes = function createRoutes ({
         }
       }
     })
+    if (typeof trailingSlashes !== 'undefined') {
```

Comment:
I think for consistency with other parts, this would be better to handled/normalized in config package. This way, during startup all modules access to correct resolved value.

Reply:
I'm afraid it's not that easy because I need access to `route` (to set page options). There is no normalization that is global / doesn't need the route access.

Reply:
Oops sorry, i wrongly read next line. That's the totally correct place